### PR TITLE
feat(tabs): add Tabs component and split payments and tips screens

### PR DESCRIPTION
**Pull request by etharner**
This PR implements and uses Tabs component which is needed for rebranding of Send/Receive, Tip/Claim and AENS

### CODE REVIEW

**Review by davidyuk — CHANGES_REQUESTED** [review]
The suggested solution looks interesting, but I'm proposing to use a straightforward router-based solution instead.

**Comment by davidyuk** [comment]
Screenshot 2021-04-22 at 17 41 47

probably we need to remove extra account info on payments-receive page

**Review by davidyuk — APPROVED** [review]

File: src/popup/router/pages/Tips.vue [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <BalanceInfo />
+    <Tabs>
+      <RouterLink
+        :to="{ name: 'tips-send' }"
+        exact-path
+      >
+        <TipIcon /> {{ $t('pages.titles.send-tip') }}
+      </RouterLink>
+      <RouterLink :to="{ name: 'tips-claim' }">
+        <ClaimTipsIcon /> {{ $t('pages.titles.claim-tips') }}
```
Comment (imollov) [inline src/popup/router/pages/Tips.vue:13]: Icon seems to be different from designs in Figma, as well as letter casing:

Screenshot 2021-04-26 at 11 29 36

File: src/popup/router/components/Header.vue [hunk 2]
```diff
@@ -121,8 +121,10 @@ export default {
     ...mapMutations(['setNotificationsStatus']),
     back() {
       const fallBackRoute = this.isLoggedIn ? '/account' : '/';
+      let { fullPath } = this.$route;
+      fullPath = fullPath.endsWith('/') ? fullPath.slice(0, -1) : fullPath;
```
Comment (imollov) [inline src/popup/router/components/Header.vue:125]: Isn't it a bad practice to mutate router object props?

Comment (etharner) [reply src/popup/router/components/Header.vue:125]: I'm making destructuring assignment of string, it'll just copy the value

**Review by davidyuk — APPROVED** [review]

**Pull request merged**